### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by PhoneDroid** [harttle/liquidjs#574] [Working with Shopify]
Hi, I was looking for some way to properly parse any 
Shopify liquid files, modify an AST and strinfigy it
to liquid again.

When I tried parsing some liquid files I got stuck on
some tags that seemingly aren't supported, `style` and
`javascript`.

How well is liquidjs compatible with Shopify and 
are there any plans to be completely compatible?

I was hoping to work with it like with BabelJS ..

**Comment by harttle** [comment]
Please take a look at https://github.com/harttle/liquidjs/discussions/528 and https://liquidjs.com/tutorials/differences.html

Furthermore, this repo doesn't contain features only provided on Shopify website (not in shopify/liquid repo).

**Reply by PhoneDroid** [reply]
So it would be possible with plugins but there aren't ones for all features it seems .. ?

**Reply by harttle** [reply]
Yes, you can define some tags to make it pass but there's no such plugin yet.
